cypress-tests/cypress/e2e/RoutingTest/00000-AccountCreate.cy.js

```diff
@@ -0,0 +1,24 @@
+import apiKeyCreateBody from "../../fixtures/create-api-key-body.json";
+import merchantCreateBody from "../../fixtures/merchant-create-body.json";
+import State from "../../utils/State";
+
+let globalState;
+describe("Account Create flow test", () => {
+  before("seed global state", () => {
+    cy.task("getGlobalState").then((state) => {
+      globalState = new State(state);
+      console.log("seeding globalState -> " + JSON.stringify(globalState));
```

Comment:
Can you please remove all the console logs that exposes `globalState`.  
That includes `cy.task("cli_log")` stuffs as well. Let's keep that them only for debugging.